### PR TITLE
SAK-45353: Samigo: Current workflow for answer markers for fill in the blank questions is confusing

### PR DESCRIPTION
**Pull request by FerAlvarez95**

### CODE REVIEW

File: samigo/samigo-app/src/webapp/jsf/author/item/fillInTheBlank.jsp [hunk 1]
```diff
@@ -39,8 +39,19 @@
           $(function () {
               defining_answers = $("#defining_answers").html();
               mutually_exclusive = $("#mutually_exclusive").html();
-              checkMarkers();
-              $("#itemForm\\:markers").change(function () {
+              var markers = $("#itemForm\\:newmarkers").val();
```
Comment (adrianfish) [inline samigo/samigo-app/src/webapp/jsf/author/item/fillInTheBlank.jsp:42]: I'd use a const here. I've added eslint to the webcomponents project and banned vars there as they are brittle, so we may as well start using const where we can.

File: samigo/samigo-app/src/webapp/jsf/author/item/fillInTheBlank.jsp [hunk 2]
```diff
@@ -49,8 +60,24 @@
               return str.replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/>/g, '&gt;');
           }
 
+          function markerRadio() {
+              var markerState = $("input[name='itemForm\\:customMarker']:checked").val();
```
Comment (adrianfish) [inline samigo/samigo-app/src/webapp/jsf/author/item/fillInTheBlank.jsp:64]: Use a const here

File: samigo/samigo-app/src/webapp/jsf/author/item/fillInTheBlank.jsp [hunk 3]
```diff
@@ -49,8 +60,24 @@
...
+                  $("#customMarkerSettings").slideUp();
+                  last_markers = $("#itemForm\\:newmarkers").val();
+                  $("#itemForm\\:newmarkers").val("{}");
+                  checkMarkers();
+              }
+          }
+
           function checkMarkers() {
-              var markerPair = $("#itemForm\\:markers").val();
+              var markerPair = $("#itemForm\\:newmarkers").val();
```
Comment (adrianfish) [inline samigo/samigo-app/src/webapp/jsf/author/item/fillInTheBlank.jsp:80]: again, const :)

**Review by adrianfish — APPROVED** [review]
Some really small suggestions around using const instead of var. Nice change!

**Comment by FerAlvarez95** [comment]
Thank you for your suggestions! All done!

**Comment by adrianfish** [comment]
Thanks. When it's built, get it merged in and we can have it verified.

**Pull request merged**